<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileArchiveOptionsClass</class>
 <widget class="QWidget" name="FileArchiveOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>45</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chbDatabaseSync">
     <property name="text">
      <string>Synchronize archive database at startup</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chbLocation">
       <property name="text">
        <string>Move history to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lneLocation">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tlbLocation">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
